src/App.js

```diff
@@ -1,25 +1,93 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { useEffect, useState } from "react";
+import { Selector } from "./components/Selector";
+import { CalculatorWindow } from "./components/CalculatorWindow";
+import { stateTable } from "./stateData";
+import { ComparisonOptions } from "./apps/RatioComparer/ComparisonOptions";
+import Player from "./apps/Player/Index";
+import TopBar from "./components/TopBar";
+import About from "./components/About";
+import { playerStateData } from "./apps/Player/playerStateData";
+import BottomBar from "./components/BottomBar";
 
 function App() {
+  const [stateData, setStateData] = useState(stateTable);
+  const [playerState, setPlayerState] = useState(playerStateData);
+
+  // useEffect(() => {
+  //   console.log({ stateData });
+  // }, [stateData]);
+
+  const [width, setWidth] = useState();
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth - 1);
+      setWidth(window.innerWidth);
+    };
+
+    window.addEventListener("resize", handleResize);
+
+    return (_) => {
+      window.removeEventListener("resize", handleResize);
+    };
+  });
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div id="Top" className="container">
+      <TopBar stateData={stateData} setStateData={setStateData} />
+      <div className="mainDiv" id="mainDiv">
+        {stateData.about.showAbout && (
+          <About stateData={stateData} setStateData={setStateData} />
+        )}
+        {!stateData.about.showAbout && (
+          <div>
+            <Player
+              stateData={stateData}
+              setStateData={setStateData}
+              playerState={playerState}
+              setPlayerState={setPlayerState}
+            />
+            <h1>Scale Creation Tools</h1>
+            <table>
+              <tbody>
+                <tr>
+                  <td>
+                    <Selector
+                      stateData={stateData}
+                      setStateData={setStateData}
+                    />
+                  </td>
+                  <td>
+                    <ComparisonOptions
+                      stateData={stateData}
+                      setStateData={setStateData}
+                    />
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+            <CalculatorWindow
+              stateData={stateData}
+              setStateData={setStateData}
+            />
+            <br />
```

Comment:
I'd try using CSS to apply margins rather than adding `<br />` elements directly - it's no longer recommended to use this element for layout.

> [br](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element) elements must be used only for line breaks that are actually part of the content, as in poems or addresses.

https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-br-element